index.js

```diff
@@ -82,6 +82,24 @@ exports.uint32 = {
   }
 }
 
+exports.float = {
+  preencode (state, n) {
+    state.end += 8
+  },
+  encode (state, n) {
+    const float = new Float64Array([n])
```

Comment:
Is there a DataView method for this instead? Then we can enforce little endian through that.
Also I think we should name it float64